.github/workflows/build.yml

```diff
@@ -0,0 +1,290 @@
+# https://help.github.com/en/categories/automating-your-workflow-with-github-actions
```

Comment:
Rebase onto `master` when #594 is merged, @ondrejmirtes pointed out that should get the build to run in this PR

Reply:
#594 is merged, please could you rebase now?

Reply:
Rebased.